pkg/packet/bgp/prefix_sid.go

```diff
@@ -540,3 +543,135 @@ func (s *SRv6SIDStructureSubSubTLV) String() string {
 		s.TranspositionOffset,
 	)
 }
+
+func NewPathAttributePrefixSID(psid *api.PrefixSID) (*PathAttributePrefixSID, error) {
+	t := BGP_ATTR_TYPE_PREFIX_SID
+	s := &PathAttributePrefixSID{
+		PathAttribute: PathAttribute{
+			Flags: PathAttrFlags[t],
+			Type:  t,
+		},
+		TLVs: make([]PrefixSIDTLVInterface, 0),
```

Comment:
is it possible for `TLVs` to be just `nil` by default?

Reply:
what is the reason to have it nil instead of allocated but with 0 elements?

Reply:
i've taken a look to what it compiles to: it has different variants for `nil` slice and `make([]T, 0)`: the latter really calls `runtime.makeslice` https://godbolt.org/z/W2_xKd . But I've benchmarked it, and there is no noticable difference, so be it. Sorry for inconvinience :)